Makefile

```diff
@@ -208,6 +197,22 @@ build/report.html: gecko.owl | build/robot.jar
 	--output $@
 
 
+### COGS Set Up
+
+BRANCH := $(shell git branch --show-current)
+
+init-cogs: .cogs
+
+# required env var GOOGLE_CREDENTIALS
+.cogs: | $(TEMPLATES)
+	cogs init -u $(EMAIL) -t "GECKO $(BRANCH)" $(foreach T,$(TEMPLATES), && cogs add $(T) -r 1)
```

Comment:
With the template rows restored, this should be `cogs add $(T) -r 3`?

Reply:
Just `-r 2` because we don't have a validation row right now, but I've fixed this.